vm-setup/roles/virtbmc/defaults/main.yml

```diff
@@ -1,7 +1,8 @@
 # Can be set to "teardown" to destroy a previous configuration
 virtbmc_action: setup
 sushy_ignore_boot_device: False
-sushy_vmedia_verify_ssl: True
+# This matches how real hardware behaves
+sushy_vmedia_verify_ssl: False
```

Comment:
I can see that `SUSHY_EMULATOR_VMEDIA_VERIFY_SSL` is now [defaulting](https://github.com/openstack/sushy-tools/blob/569a02c1ccca1a6824fc0b9542ac2966bcaf997a/doc/source/admin/emulator.conf#L100 ) to `False`  to _match the actual bare metal hardware_, but I'm just curious why real hardware would not do host verification before pulling the image?

Reply:
Because until very recently there was no way to provide custom certificates. And without custom certificates this feature cannot be reasonably used.